account/tests/test_professional.py

```diff
@@ -0,0 +1,69 @@
+from account.models.profile import Profile
+from account.models.professional import Professions, Professional
+from .test_profile import CITY
+from .test_profile import FIRST_NAME
+from .test_profile import LAST_NAME
+import pytest
+
+
+PROFESSION = Professions.Locksmith
+DESCRIPTION = 'I'
+
+
+@pytest.fixture
+def save_professionals(professional, professional2, professional3):
+    professional.profile_id.user_id.save()
+    professional.profile_id.save()
+    professional.save()
+    professional2.profile_id.user_id.save()
+    professional2.profile_id.save()
+    professional2.save()
+    professional3.profile_id.user_id.save()
+    professional3.profile_id.save()
+    professional3.save()
```

Comment:
i think this needs to be in `conftest.py`

Reply:
@guybeckenstein Done